tests/src/e2e/ledger_tests.rs

```diff
@@ -946,27 +952,6 @@ fn pos_bonds() -> Result<()> {
 
     let validator_0_rpc = get_actor_rpc(&test, &Who::Validator(0));
 
-    // put money in the validator account from its balance account so that it
```

Comment:
nice that we don't have to do this anymore

Reply:
yeah for sure